comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -112,7 +112,8 @@ fi
 # build a list of JVM arguments
 
 if [[ $LIBDIR ]]; then
-  JVMOPTIONS="${JVMOPTIONS} -classpath ${LIBDIR}"
+  JVMOPTIONS="${JVMOPTIONS} -classpath"
```

Comment:
I'm not sure I follow the logic here: why not include the LIBDIR value immediately after the -classpath option?

Reply:
This is what happens if I use the original line 115:
```bash
$ ~/src/Co\ Mixed/bin/run.sh -v
+ [[ ! -f /Users/jfine/src/Co Mixed/bin/comixed-app-0.8.2-1.1.jar ]]
+ false
+ false
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ -n /Users/jfine/src/Co Mixed/bin/../lib ]]
+ JVMOPTIONS=' -classpath /Users/jfine/src/Co Mixed/bin/../lib'
+ /usr/bin/java -classpath /Users/jfine/src/Co Mixed/bin/../lib -jar '/Users/jfine/src/Co Mixed/bin/comixed-app-0.8.2-1.1.jar'
Error: Could not find or load main class Mixed.bin....lib
```
I've not had this problem with Linux before, so I wonder if it's a weird Mac bash quirk.  (Mojave uses bash 3.2.x)

My workaround was to put the LIBDIR (which contains a space in it) all in its variable so I can properly shell escape.  I created a second CLASSPATH variable so if LIBDIR doesn't exist (but was set), no string is substituted.

Reply:
Tested this on a Linux machine with bash 4.1.x -- same problem.